shell.nix

```diff
@@ -0,0 +1,19 @@
+{ java ? "openjdk11" }:
```

Comment:
Should it be the default java version for everything in the environment? If it should be, then:

- Why is openjdk8 installed too?
- I think you should override jdk/java version with this one for `metals` and `sbt`. If not, they are going to use the default ones in nixpkgs.

Reply:
Done 2d6572b

Reply:
8 was installed because of @pepegar not deleting that before :D